zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/authorization/PersistedMapping.java

```diff
@@ -77,4 +81,22 @@ public PersistedMapping addRoleKey(final long roleKey) {
     roleKeysProp.add().setValue(roleKey);
     return this;
   }
+
+  public List<String> getTenantIdsList() {
```

Comment:
🙃 We can only name this `getTenantIds()` as it makes it more consistent with the setter.

Reply:
I prefer to use this naming convention, it's consistent with other Persisted record (eg. PersistedUser). I'll rather change the set name 👍